example/config.yml

```diff
@@ -1,6 +1,7 @@
 backend:
-  name: test-repo
```

Comment:
Assuming this was the plan, but we should remove the example changes before merging.